conans/client/profile_loader.py

```diff
@@ -261,6 +261,12 @@ def get_profile(profile_text, base_profile=None):
         options = Options.loads(doc.options) if doc.options else None
         tool_requires = _ProfileValueParser._parse_tool_requires(doc)
 
+        if doc.system_tools:
+            system_tools = [RecipeReference.loads(r.strip())
```

Comment:
For anyone wondering as I did if this could be improved, not until we support 3.8 with https://peps.python.org/pep-0572/ , which would let us write something like:
```py
system_tools = [RecipeReference.loads(stripped) for r in doc.system_tools.splitlines() if (stripped := r.strip())]
```